recipe/meta.yaml

```
@@ -1,23 +1,38 @@
+# Name for dataset. User chosen.
 title: "Title for your recipe"
-description: "Long description for your recipe"
+# Description of dataset.  User chosen, roughly 1 sentance in length.
+description: "Long description for your recipe" 
+# Version of pangeo_forge_recipes library that was used 
 pangeo_forge_version: "0.8.0"
```

```suggestion
pangeo_forge_version: "0.8.2"
```

One more suggestion; let's make this default to `0.8.2` because that's what's installed in the sandbox environment.